### DISCUSSION TITLE
Can I do a full backup where certain files are treated as unchanged from the last backup version, even if they have changed?

### DISCUSSION THREAD

**Discussion by lmfont** [borgbackup/borg#7831] [Q&A]
I use Borg to store a complete backup of my home directory. I would really like to do all of the following, simultaneously:

 * Run a backup every day to achieve file versioning
 * For a specific list of directories, only back them up on request (e.g. once a month). These directories contain large, non-compressible files that change frequently, but the full history of the file is not important.
 * Be able to restore all my files from the most recent backup without having to manually pull and compare from multiple archives.
 * Be able to purge old backups without having to worry about which ones are "full" and which ones are "partial"

Basically, to boil it down, I want to be able to do backups that are effectively full backups, but treat files on a specific list as unchanged even if they have changed.

Specifically, changing the included / excluded files does *not* seem to do what I want. E.g. if I include only ~/Documents, and then mount that archive version, I see only files in ~/Documents. That's not good because (a) it means I can't restore all my files from this backup, and (b) if I purge the wrong previous backup, I risk losing data.

I'd also be interested in answers talking about Borg2, if applicable.

**Comment by ThomasWaldmann** [comment]
Well, the usual way to solve this is to just consider that stuff to be 2 different backups of 2 different data sets.

One would then use borg to create 2 differently named series of backup archives, have different schedules to run borg create for each dataset and possibly also different borg prune settings.

It's somehow similar to creating a backup of system and user files:
- user files are very important and one wants many backups (frequently in recent past, less frequently in far past)
- system files do not need to be kept that long and also could be backed up less frequently.

At restore time, one usually would restore the latest system backup and then the latest user file backup.

I think that's the preferable way of doing things. Implementing the behaviour as you described would make borg even more complex and harder to understand than it already is.

**Comment by ThomasWaldmann** [comment]
Closing this as outdated: it has seen no activity for a long time and predates the current borg releases.

If this still affects you on a current version - borg 1.4.x (stable) or the borg 2.0.0 beta - please open a new discussion with up-to-date details.
